CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ Main (unreleased)
   - `discovery.openstack` - service discovery for OpenStack. (@marctc)
   - `discovery.hetzner` - service discovery for Hetzner Cloud. (@marctc)
   - `loki.write` now exposes basic WAL support. (@thepalbi)
+  - `loki.write` WAL now exposes a last segment reclaimed metric. (@thepalbi)
```

Comment:
nit: Could we move these two log entries to "Enhancements" instead of below "New Grafana Agent Flow components".

Reply:
Yess

Reply:
Could we also move the previous entry, since it's not a new component?

Reply:
Donee